src/core/core_structs/results.jl

```diff
@@ -1,12 +1,12 @@
-struct OperationModelResults
+struct OperationsProblemResults
     variables::Dict{Symbol, DataFrames.DataFrame}
     total_cost::Dict{Symbol, Any}
     optimizer_log::Dict{Symbol, Any}
     time_stamp::DataFrames.DataFrame
 
 end
 
-function get_variable(res_model::OperationModelResults, key::Symbol)
+function get_variable(res_model::OperationsProblemResults, key::Symbol)
         try
```

Comment:
This looks like a bug.  The return statement should be inside the try block.

Reply:
Should be fixed now 
6675339